specs/experimental/plasma.md

```diff
@@ -83,6 +83,16 @@ on the `commitment_type_byte` where [0, 128) are reserved for official implement
 | `commitment_type` | `commitment`                    |
 | ----------------- | ------------------------------- |
 | 0                 | `keccak256(tx_payload)`         |
+| 1                 | `da-server`                     |
+
+The `da-server` commitment is as follows: `version_byte ++ len(payload) as uvarint ++ payload`. The version byte
```

Comment:
I am confused when you say `version_byte` - there is a table above that defines `version_byte`. Can we call it `commitment_version_byte` to make it specific and different from the previous definition?

Reply:
Will rename to try to clarify. We have lots of version bytes so will specify the purpose of each one